bin/crew

```diff
@@ -1130,53 +1091,40 @@ def archive_package (pwd)
 end
 
 def remove (pkgName)
-
-  #make sure the package is actually installed
-  unless @device[:installed_packages].any? { |pkg| pkg[:name] == pkgName } || File.exist?("#{CREW_META_PATH}#{pkgName}.filelist")
-    puts "Package #{pkgName} isn't installed.".lightred
-    return
-  end
-
-  #if the filelist exists, remove the files and directories installed by the package
-  if File.file?("#{CREW_META_PATH}#{pkgName}.filelist")
-    Dir.chdir CREW_CONFIG_PATH do
-
-      #remove all files installed by the package
-      File.open("meta/#{pkgName}.filelist").each_line do |line|
-        begin
-          puts 'Removing file ' + line.chomp + ''.lightred if @opt_verbose
-          File.unlink line.chomp
-        rescue => exception #swallow exception
-        end
-      end
-
-      #remove all directories installed by the package
-      File.readlines("meta/#{pkgName}.directorylist").reverse.each do |line|
-        begin
-          puts 'Removing directory ' + line.chomp + ''.lightred if @opt_verbose
-          Dir.rmdir line.chomp
-        rescue => exception #swallow exception
-        end
-      end
-
-      #remove the file and directory list
-      File.unlink "meta/#{pkgName}.filelist"
-      File.unlink "meta/#{pkgName}.directorylist"
-
+  # make sure the package is actually installed
+  unless installed?(pkgName, mode: 'filelist')
+    return puts "Package #{pkgName} isn't installed.".lightred
```

Comment:
Why are returning a value when this isn't in a function?

Reply:
The `return` block is used to jump out the function (it will return `nil` as `puts` will returns nothing)

Reply:
But this isn't a function.

Reply:
> This line is under `def remove`